FirebaseDatabase/Sources/Utilities/FUtilities.h

```diff
@@ -28,6 +28,7 @@
 + (NSError *)errorForStatus:(NSString *)status andReason:(NSString *)reason;
 + (NSNumber *)intForString:(NSString *)string;
 + (NSString *)ieee754StringForNumber:(NSNumber *)val;
++ (BOOL)tryParseStringToInt:(NSString *)string asInt:(NSInteger *)integer;
```

Comment:
The method name is `tryParseStringToInt: asInt:`. I would suggest removing the `ToInt` suffix.